Frameworks/include/MockClass.h

```diff
@@ -529,12 +529,12 @@ namespace Mock {
 template <typename TFunctor>
 void ValidateCallback(const TFunctor& functor, const wchar_t* name) {
     WEX::TestExecution::SetVerifyOutput verifySettings(WEX::TestExecution::VerifyOutputSettings::LogOnlyFailures);
-    VERIFY_IS_NOT_NULL(functor, WEX::Common::String().Format(L"The callback for %s has not been set!", name));
+    VERIFY_IS_NOT_NULL(functor, WEX::Common::String().Format(L"The callback for %ls has not been set!", name));
 }
-#elif defined(ASSERT_NE_MSG)
+#elif defined(ASSERT_TRUE_MSG)
```

Comment:
this is the easiest way to prefer gtest if present but not break back compat

---
In reply to: [82296551](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82296551) [](ancestors = 82296551,82293757)